<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生选课系统</string>
  </property>
  <widget class="QLabel" name="lbtime">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_inf">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>个人信息(I)</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_left">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>111</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>我的课表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>可选课程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Nselect">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>退选课程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_refresh1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_refresh2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_right">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>111</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pb_search">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_select">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>选择课程</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_status">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>系统开放选课中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_logout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登出(q)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
